gbfs-validator/__test__/gbfs.v3.0-RC.test.js

```diff
@@ -0,0 +1,148 @@
+const GBFS = require('../gbfs')
+
+function get_errors(result) {
+  let errors = []
+
+  result.files?.map(f => {
+    if (f.errors) {
+      errors.push({ file: f.file, errors: f.errors })
+    }
+
+    f.languages?.map(l => {
+      if (l.errors) {
+        errors.push({ file: f.file, lang: l.lang, errors: l.errors })
+      }
+    })
+  })
+
+  return errors
+}
+
+describe('default feed', () => {
+  let gbfsFeedServer
+
+  beforeAll(async () => {
+    const { MockRequests } = require('./fixtures/v3.0-RC/default')
+    let mockRequests = new MockRequests()
+
+    gbfsFeedServer = mockRequests.build()
+
+    await gbfsFeedServer.listen()
```

Comment:
Merci @tdelmas @PierrickP pour votre patience.

Lorsque j'ai roulé les tests en local, ceux de v3.0 ne passaient pas à cause d'une erreur `Invalid URL`.

Pour que les tests de v3.0 passent avec succès, j'ai du spécifier l'adresse du serveur, comme pour les tests des versions précédentes : https://github.com/MobilityData/gbfs-validator/blob/b8e6df110e79a335f6c0d0ba2d66007e6efdd37b/gbfs-validator/__test__/gbfs.test.js#L3-L6

Et l'ajouter à chaque test de v3.0, comme pour les tests des versions précédentes : https://github.com/MobilityData/gbfs-validator/blob/b8e6df110e79a335f6c0d0ba2d66007e6efdd37b/gbfs-validator/__test__/gbfs.test.js#L63

Quelle serait la prochaine étape pour effectuer ce changement dans les tests de v3.0 ?

Merci

Reply:
C'est une erreur spécifique à Windwons, fixé sur https://github.com/MobilityData/gbfs-validator/pull/104/commits/33be4864114bceb75d6c660066018992097e9019

Reply:
Thank you both. This may also be a bug on Mac as I'm on macOS 13.4 and got the error.